plugins/UsersManager/templates/userSettings.twig

```diff
@@ -107,6 +107,15 @@
     </form>
 </div>
 
+<div piwik-content-block
+     content-title="{{ 'UsersManager_TokenAuth'|translate|e('html_attr') }}">
+    <p id="token_auth_user" data-token="{{ userTokenAuth }}">{{ userTokenAuth }}</p>
```

Comment:
Do we need the token in the attribute again or could we just read `$('#token_auth_user').text()`? Not important though I think